Octokit.Tests/Reactive/ObservableCheckRunsClientTests.cs

```diff
@@ -178,7 +178,7 @@ public async Task RequestsCorrectUrlWithRequest()
                             && x["check_name"] == "build"
                             && x["status"] == "in_progress"
                             && x["filter"] == "latest"),
-                    "application/vnd.github.antiope-preview+json");
+                    null);
```

Comment:
I think instead of passing null, now that previews are gone and will be able to use the default content type / accept headers set by the REST API we should consider creating an overload for 

`Task<IApiResponse<T>> Get<T>(Uri uri, IDictionary<string, string> parameters);` 

Then in the implementation we simply pass in null there for the `accepts` parameter:

```
     public Task<IApiResponse<T>> Get<T>(Uri uri, IDictionary<string, string> parameters)
        {
            Ensure.ArgumentNotNull(uri, nameof(uri));

            return SendData<T>(uri.ApplyParameters(parameters), HttpMethod.Get, null, null, null, CancellationToken.None);
        }
```

That way, we give users the ability to use the defaults and the option to pass in headers if they'd like.  This aspect of the code could be refactored and allow a collection of headers to be passed in, but that's orthogonal to what you're doing here.

It also cleans up call sites a bit so that we're not passing nulls around until we absolutely have to.

Thoughts?

Reply:
I've made some tweaks, can you have another skim over?